systemtest/pom.xml

```diff
@@ -229,6 +229,11 @@
             <groupId>io.fabric8</groupId>
             <artifactId>openshift-model-operatorhub</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.skodjob</groupId>
+            <artifactId>test-frame-metrics-collector</artifactId>
+            <version>0.1.0</version>
```

Comment:
use property instead. and is it ok to include directly in systemtest module instead of root?

Reply:
yeah, will update this thanks :). 

> include directly in systemtest module
sure, thanks

Reply:
Updated :)